internal/testutil/test_hcls.go

```diff
@@ -570,6 +580,41 @@ func RHclBackboneVirtualCircuitVlan() RHclBackboneVirtualCircuitResult {
 	}
 }
 
+// packetfabric_port_loa
+func RHclPortLoa() RHclPortLoaResult {
+
+	c, err := _createPFClient()
+	if err != nil {
+		log.Panic(err)
+	}
+
+	portDetails := PortDetails{
+		PFClient:          c,
+		DesiredSpeed:      portSpeed,
+		IsCloudConnection: true,
+	}
+
+	hclPortResult := portDetails.RHclPort(false)
+	resourceName, hclName := GenerateUniqueResourceName(pfPortLoa)
+
+	hcl := fmt.Sprintf(RResourcePortLoa,
+		hclName,
+		hclPortResult.ResourceReference,
+		PortLoaCustomerName,
+		PortLoaDestinationEmail,
```

Comment:
Let's use an environment variable for this `PF_USER_EMAIL`. Will have to add it into Jenkins too,